SnowSQL/public/Open-SnowSqlConnection.ps1

```diff
@@ -42,10 +53,24 @@ function Open-SnowSqlConnection
             Query       = '!help'
             ErrorAction = 'Stop'
             Connection  = $SnowSqlConnection
+            timeout     = $Timeout
+        }
+        if ($PSCmdlet.ShouldProcess("Execute SnowSql query [$($invokeSnowSqlSplat.query)] on [$Endpoint] as [$($Credential.UserName)]. Use -Debug to see full command"))
+        {
+            $Result = Invoke-SnowSql @invokeSnowSqlSplat
+        }
+        else
+        {
+            $Result = $true
+        }
+        if (-not $Result)
+        {
+            Write-Error ("Unable to connect to SnowSql endpoint {0}" -f $Endpoint) -ErrorAction Stop
```

Comment:
I'm not a fan of intentionally swallowing original error output. I would rather see credential information explicitly replaced with ****** in output or something if there is something you are trying to hide. I would like to know why SnowSQL rejected my connection.

Reply:
OK ! It's fix.
![image](https://user-images.githubusercontent.com/25510477/93722640-b9db9e00-fb98-11ea-980a-90aa7ef38320.png)